installer/macos/scripts/postinstall

```diff
@@ -50,3 +50,5 @@ mkdir -m 777 -p $MASSASTATION_CERT_DIR || fatal "certs directory creation failed
 ping -c 1 -t 1 test.massa >/dev/null 2>&1 || set_local_dns
 
 mv ./uninstall.sh $MASSASTATION_CONFIG_DIR || fatal "uninstall script move failed."
+
+LOG_LEVEL=DEBUG $2/MassaStation.app/Contents/MacOS/massastation --repair || fatal "failed to repair MassaStation"
```

Comment:
Are you sure debug is needed? Can it disturb the enduser?

Reply:
The end user in the nominal scenario won't see the logs. I chose to put DEBUG level here to ease debugging when we will potentially ask for the logs to a user.

In the repair process, many logs are in DEBUG level.